### PR TITLE
Add privacy policy

### LINKED ISSUE #79: Privacy Policy / Code of Conduct
Add a "policies" github repo which has our privacy policy / terms of conduct

Also update the questions in the application portal to link to our policies instead of MLH

Also update links in the main website for this

### PR DESCRIPTION
**Pull request by theopolisme**
Fixes #79

### CODE REVIEW

File: src/FormPage/FormPageWrapper.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ class FormPageWrapper extends React.Component<IFormPageWrapperProps, { showSaved
...
         }
         if (get(schema, 'properties.accept_share')) {
-            schema.properties.accept_share.title = <span>I authorize you to share my application/registration information for event administration, ranking, MLH administration, pre- and post-event informational e-mails, and occasional messages about hackathons in-line with the <a className="form-link" href="https://mlh.io/privacy" target="_blank" onClick={e => e.stopPropagation()}>MLH Privacy Policy</a>. I further I agree to the terms of both the <a className="form-link" href="https://github.com/MLH/mlh-policies/blob/master/prize-terms-and-conditions/contest-terms.md" target="_blank" onClick={e => e.stopPropagation()}>MLH Contest Terms and Conditions</a> and the MLH Privacy Policy.</span>;
+            schema.properties.accept_share.title = <span>I authorize you to share my application/registration information for event administration, ranking, pre- and post-event informational e-mails, and occasional messages about hackathons in-line with the <a className="form-link" href="https://www.treehacks.com/privacy-policy" target="_blank" onClick={e => e.stopPropagation()}>TreeHacks Privacy Policy</a>.</span>;
```
Comment (epicfaace) [inline src/FormPage/FormPageWrapper.tsx]:
```suggestion
            schema.properties.accept_share.title = <span>I authorize you to share my application/registration information for event administration, ranking, pre- and post-event informational e-mails, and occasional messages about hackathons in-line with the <a className="form-link" href="https://github.com/TreeHacks/policies/blob/master/privacy-policy.md" target="_blank">TreeHacks Privacy Policy</a>.</span>;
```

Comment (epicfaace) [inline src/FormPage/FormPageWrapper.tsx]: Maybe we should remove "for event administration, ranking, pre- and post-event informational e-mails, and occasional messages about hackathons"? We don't specifically talk about these things in the actual privacy policy, so maybe it's best to just refer them to the privacy policy page instead.

Comment (theopolisme) [reply src/FormPage/FormPageWrapper.tsx]: Sounds good to me

Comment (theopolisme) [reply src/FormPage/FormPageWrapper.tsx]: I think we still need to stopPropagation otherwise leads to a phantom click

**Pull request merged**